examples/long-running/long-running-script.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use DDTrace\SpanData;
+use DDTrace\Type;
+
+// This part is required for long running processes
+\DDTrace\trace_function('repetitive_function', function (SpanData $span, $args) {
+    $span->service = getenv('DD_SERVICE');
+    $span->name = $span->resource = 'repetitive_function';
+    $span->type = Type::CLI; // or Type::MESSAGE_PRODUCER or anything appropriate from DDTrace\Type::
+});
```

Comment:
What do you think about this?
```suggestion
\DDTrace\trace_function('repetitive_function', function (SpanData $span, $args) {
    /* Optional: $span->name and $span->resource
     * `name` defaults to the function/method name being traced.
     * `resource` defaults to span's name. If you set it, use something with low cardinality.
     */
    
    // Required:
    $span->service = getenv('DD_SERVICE');
    $span->type = Type::CLI; // or Type::MESSAGE_PRODUCER or anything appropriate from DDTrace\Type::
});
```